sdks/python/setup.py

```diff
@@ -286,6 +286,7 @@ def get_portability_package_data():
             'pytest-xdist>=2.5.0,<3',
             'pytest-timeout>=2.1.0,<3',
             'scikit-learn>=0.20.0',
+            'tensorflow>=1.0.0',
```

Comment:
let's not depend on a large dependency like this by default. you can see how we deal with sklearn/pytorch in  similar circumstances.

Reply:
+1 to @tvalentyn.

Reply:
Commented at https://github.com/apache/beam/issues/25547#issuecomment-1435716734. 

We can follow the approach of sklearn/torch/tensorRT gradle tests and not add very large ML dependencies to the setup.py

Reply:
Ah, I see - so this was running on 3.7 and 3.9, just not 3.8 because of the difference in dependencies here - https://github.com/apache/beam/blob/921bc7b94708391eb7d980cb8d9ed35c80d3aebb/build.gradle.kts#L457

Thanks for clarifying. I'm now +1 on removing this and manually triggering Python 3.7 postcommits, sorry for my confusion @riteshghorse 

---------------------------

A couple of follow up questions for @AnandInguva and @tvalentyn :
1) Is there any reason we only run the dataflow versions on 3.8? Since TensorRT is on Dataflow, we now need to run both the 3.7 and 3.8 postcommits to fully exercise our inference code base
2) Is there any reason we're not running most of our inference postcommits on dataflow (as well as the direct runner)? That's probably where most of our usage is today